go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/hashicorp/vault/api v1.0.2
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/influxdata/cron v0.0.0-20191203200038-ded12750aac6
-	github.com/influxdata/flux v0.73.0
+	github.com/influxdata/flux v0.73.1-0.20200714212633-e1dbd2426587
```

Comment:
Just so you know, we can't merge this to master as it is now, since `influxdb` must point at a Flux release. So we'll need to wait until flux is released again, and update this `go.mod` file to use the new release.